Added clarifying language for the LIBRARY_NAME (#16611)
aspnet/AspNetCore.Docs: aspnetcore/razor-pages/ui-class.md

### Consume content from a referenced RCL

> The files included in the *wwwroot* folder of the RCL are exposed to the consuming app under the prefix `_content/{LIBRARY NAME}/`. For example, a library named *Razor.Class.Lib* results in a path to static content at `_content/Razor.Class.Lib/`.  In cases where the assembly name is not the same as the package id, the `{LIBRARY NAME}` will reference the package id instead.

Reviewer: ```suggestion
The files included in the *wwwroot* folder of the RCL are exposed to the consuming app under the prefix `_content/{LIBRARY NAME}/`. For example, a library named *Razor.Class.Lib* results in a path to static content at `_content/Razor.Class.Lib/`. When producing a NuGet package and the assembly name isn't the same as the package ID, use the the package ID for `{LIBRARY NAME}`.
```

Thanks for sending this in @csharpfritz. Just nits ...

* One space between sentences.
* "In cases ..." ... this is just for NuGet packages, correct?
* Contraction: "is not" :point_right: "isn't"
* Capitalize abbreviation "id" ... IIRC, the NuGet folks write "package ID" in their docs ... *I think* 🤔
* Present tense "will reference" :point_right: "references"
* Recommend imperative tone: Tell the dev exactly what to do ... use the package ID.

Reviewer 2: @csharpfritz  can you tap the **Commit suggestion** button so we can merge?

![image](https://user-images.githubusercontent.com/3605364/72834690-6beb5080-3c2d-11ea-813e-227099ccff49.png)
